pkg/visor/api.go

```diff
@@ -567,6 +573,31 @@ func (v *Visor) GetAppConnectionsSummary(appName string) ([]appserver.Connection
 	return nil, ErrProcNotAvailable
 }
 
+// VPNServers gets available public VPN server from service discovery URL
+func (v *Visor) VPNServers() []string {
```

Comment:
Why are we not returning an error here? This is supposed to be relayed back to the CLI invoking it I would suppose. So it should be logged there imo.

Reply:
fixed in next commit